drop-storage/sqlx-data.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is this file necessary?

Reply:
`sqlx` created it so I assumed it's somewhat needed but I can move it into gitignore I guess

Reply:
Well, it's needed in the offline case. Which probably is irrelevant to the SQLite case. I would go for removing it

Reply:
moved to gitignore and removed it